projects/kubernetes/kubernetes/1-20/patches/0022-EKS-PATCH-Update-aws-sdk-to-v1.42.23-for-ap-se-3.patch

```diff
@@ -2,9 +2,9 @@ From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
 From: Jyoti Mahapatra <jyotima@amazon.com>
 Date: Thu, 31 Mar 2022 18:28:59 -0700
 Subject: --EKS-PATCH-- Update aws-sdk to v1.42.23 for ap-southeast-3
- region build
 
 Update aws-sdk to v1.42.23 for ap-southeast-3 region build
+
```

Comment:
Was this whitespace intentional??

Reply:
Yeah it was. I probably should have clarified this in the description, but this is a change needed for the automation Bhavitha is working on